libdevcore/LruCache.h

```diff
@@ -0,0 +1,128 @@
+// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2019 Aleth Authors.
+// Licensed under the GNU General Public License, Version 3.
+
+#pragma once
+
+#include <list>
+#include <unordered_map>
+
+namespace dev
+{
+template <class Key, class Value>
+class LruCache
+{
+    typedef Key key_type;
```

Comment:
Better to use `using key_type = Key;` instead of `typedef`

Reply:
Agreed, much more intuitive / readable